cxx-checks/src/main/java/org/sonar/cxx/checks/CheckList.java

```diff
@@ -76,7 +76,8 @@ private CheckList() {
       UsingNamespaceInHeaderCheck.class,
       SafetyTagCheck.class,
       UseCorrectIncludeCheck.class,
-      XPathCheck.class,
+      // XPathCheck.class, this gives class not found exception in 5.6
+      // XPath no longer found in other plugins
```

Comment:
SQ 5.6: XML utilities org.sonar.api.utils.StaxParser and XpathParser are deprecated as plugins can now embed and use their own 3rd-party libraries
http://docs.sonarqube.org/display/DEV/API+Changes